src/main/java/org/fundacionjala/coding/german/SpinWords.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.coding.german;
+
+/**
+ * Created by Administrator on 8/29/2017.
+ */
+public class SpinWords {
+    /**
+     * @param words String
+     * @return String
+     */
+    public String spinWords(String words) {
+        String delimiter = " ";
+        String[] temp = words.split(delimiter);
+        StringBuilder temBuilder = new StringBuilder();
+        for (String s : temp) {
+            if (s.length() >= 5) {
+                temBuilder.append(new StringBuilder(s).reverse().toString());
+
+            } else {
+                temBuilder.append(s);
+            }
+            temBuilder.append(" ");
+
+        }
+        temBuilder.deleteCharAt(temBuilder.length() - 1);
+        return temBuilder.toString();
```

Comment:
Instead of using a StringBuilder adding whitespaces and removing the last whitespace.
You can use a StringJoiner.
Try this.
```java
    public String spinWords(String words) {
        String delimiter = " ";
        String[] temp = words.split(delimiter);
        StringJoiner temBuilder = new StringJoiner(delimiter);
        for (String s : temp) {
            temBuilder.add(s.length() >= 5 ? new StringBuilder(s).reverse().toString() : s);
        }
        return temBuilder.toString();
    }
```